DateTimePicker/DateTimePicker.js

```diff
@@ -67,7 +59,7 @@ class DateTimePickerBase extends React.Component {
 		this.state = {
 			hour: '12',
 			minute: '00',
-			meridiem: 'AM',
+			meridiem: $L('AM'),
```

Comment:
This change also affects the value to be delivered to the app through `onChange`, `onSave`, and `onClose` callback props. Is this intended?

Reply:
I agree with your opinion. Fixed to compare type with plane text.

Reply:
In this code, there is a problem that ilib date is not passed to App developer.
Any changes to this code will be revised in PLAT-104574.